setup.py

```diff
@@ -49,7 +49,7 @@ def read(*paths):
     "html": ["pyquery>=1.4"],
     "json": ["ijson>=3.0", "jsonlines>=1.2"],
     "ods": ["ezodf>=0.3", "lxml>=4.0"],
-    "pandas": ["pandas>=1.0"],
+    "pandas": ["pandas>=1.0", "tabulate>=0.8.9"],
```

Comment:
Let's add `tabulate` to `INSTALL_REQUIRES` - it's a common library we will use for other work too